sql_generators/active_users/templates/focus_android_view.sql

```diff
@@ -2,8 +2,8 @@ CREATE OR REPLACE VIEW
   `{{ project_id }}.{{ app_name }}.active_users_aggregates`
 AS
 SELECT
-  * REPLACE (COALESCE(country, '??') AS country),
-  `mozfun.norm.browser_version_info`(app_version) AS browser_version_info
```

Comment:
[DENG-516](https://mozilla-hub.atlassian.net/browse/DENG-516?atlOrigin=eyJpIjoiZDBlNmIzMzIyNzBmNGNmMWExMzA2OGZkZGQyNTM1NWQiLCJwIjoiamlyYS1zbGFjay1pbnQifQ) has requested for the `browser_version_info` split using this udf.  Instead of removing it, it should be added to all the views.